learning_wavelets/training_scripts/exact_recon_unet_training.py

```diff
@@ -0,0 +1,79 @@
+import os.path as op
+import time
+
+import tensorflow as tf
+from tensorflow.keras.callbacks import TensorBoard, ModelCheckpoint
+import tensorflow_addons as tfa
+
+from learning_wavelets.config import LOGS_DIR, CHECKPOINTS_DIR
+from learning_wavelets.data.datasets import im_dataset_div2k, im_dataset_bsd500
+from learning_wavelets.models.exact_recon_unet import ExactReconUnet
+
+
+
+tf.random.set_seed(1)
+
+
+def train_unet(noise_std_train=(0, 55), noise_std_val=30, n_samples=None, source='bsd500', base_n_filters=4, n_layers=4, non_linearity='relu', batch_size=8, n_epochs=50):
```

Comment:
Make this a multi-line statement.

I don't have a strict rule for line length, but this is definitely too long